actors/builtin/paych/paych_test.go

```diff
@@ -0,0 +1,179 @@
+package paych_test
+
+import (
+	"context"
+	"testing"
+
+	addr "github.com/filecoin-project/go-address"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	. "github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	"github.com/filecoin-project/specs-actors/actors/crypto"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/filecoin-project/specs-actors/support/mock"
+	tutil "github.com/filecoin-project/specs-actors/support/testing"
+)
+
+func TestPaymentChannelActor_Constructor(t *testing.T) {
+	ctx := context.Background()
+	actor := pcActorHarness{Actor{}, t}
+
+	pcaAddr := tutil.NewIDAddr(t, 100)
+	callerAddr := tutil.NewIDAddr(t, 101)
+
+	t.Run("can create a payment channel actor", func(t *testing.T) {
+		builder := mock.NewBuilder(ctx, pcaAddr).
+			WithCaller(callerAddr, builtin.AccountActorCodeID).
+			WithReceiverType(builtin.AccountActorCodeID)
+
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt, pcaAddr, callerAddr)
+	})
+
+	t.Run("fails if caller is not account actor", func(t *testing.T) {
```

Comment:
The `ExpectValidateCallerType` is sufficient that we don't need to test this. This test is really testing that the runtime correctly rejects the ValidateCallerType with the wrong caller, but that's not actually testing the actor itself.